Formula/dpkg.rb

```diff
@@ -36,7 +37,12 @@ class Dpkg < Formula
   def install
     # We need to specify a recent gnutar, otherwise various dpkg C programs will
     # use the system "tar", which will fail because it lacks certain switches.
-    ENV["TAR"] = Formula["gnu-tar"].opt_bin/"gtar"
+    on_macos do
+      ENV["TAR"] = Formula["libarchive"].opt_bin/"gtar"
+    end
+    on_linux do
+      ENV["TAR"] = Formula["libarchive"].opt_bin/"tar"
+    end
```

Comment:
```suggestion
    ENV["TAR"] = Formula["libarchive"].opt_bin/"tar"
```
`libarchive` is `keg-only`, so it has no `gtar` binary.

Reply:
It's keg-only, but only on Mac, not on linux.

Nice catch though, I fixed the path, and used `bin` instead of `opt_bin` too.